uitools/toolkitcpp.pri

```diff
@@ -13,7 +13,7 @@
 # limitations under the License.
 include($$PWD/common.pri)
 
-QT += quickcontrols2 webview svg
+QT += quickcontrols2 webview svg graphs
```

Comment:
Is CMake support coming later?

Reply:
Yes I created a separate issue for the cmake integration once everything is complete with qmake.